drake/systems/plants/RigidBody.cpp

```diff
@@ -2,6 +2,9 @@
 #include "RigidBody.h"
 #include <stdexcept>
 
+#include "drake/util/drakeGeometryUtil.h"
+#include "drake/util/testUtil.h"
```

Comment:
Waiting on PR https://github.com/RobotLocomotion/drake/pull/1953 to be merged before I can remove `#include testUtil.h`.